js/your_extension.test.js

```
@@ -0,0 +1,57 @@
+ */
+
+const fs = require('fs');
+const path = require('path');
+
+describe('Drupal.behaviors.yourExtension', () => {
+  beforeEach(() => {
+    global.Drupal = { behaviors: {} };
+
+    const filePath = path.resolve(__dirname, 'your_extension.js');
+    const code = fs.readFileSync(filePath, 'utf8');
+    eval(code);
```

Biome (2.4.4)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)